proposals/evaluation/uc-1-consent.md

```diff
@@ -0,0 +1,23 @@
+# Consent Workflow Use Cases
+
+## Agent grants access to an application to a selected subset of data across pods
+
+Alice wants to use an application called FinanceInsight to help her
+understand trends in her cashflow and spending activity, using financial
+data stored in her personal and work pods. First she'll needs to grant 
+FinanceInsight access to that data across her two pods. 
+
+Since she has a lot of other data in her pod that isn't finance related, 
+she needsto grant FinanceInsight access only to exactly the data it needs.
+
+Because there's a lot of finance data spread across her pods, along with
```

Comment:
```suggestion
Because there's a lot of finance data spread across pods she has access to, along with
```

Reply:
@elf-pavlik what about this suggestion as an alternative?

```suggestion
Because there's a lot of finance data spread across pods she either controls or has access to, along with
```

Reply:
@justinwb -- it seems to me that "pods she has access to" includes "pods she ... controls", so I wonder why the extra text is needed?

Reply:
Resolved as per https://github.com/solid/authorization-panel/pull/232